benchmarks/templates/benchmarks/leaderboard/leaderboard-table.html

```diff
@@ -0,0 +1,151 @@
+{% load static %}
+{% if models %}
+{% if has_user %}
+
+
+<form>
+{% endif %}
+
+<banner>
+  <div class="leaderboard-table-banner-icon gradient">
+      <img src="{% static "/benchmarks/img/leaderboard_header_icon.png" %}" alt="Brain-Score Logo">
+  </div>
+  <div class="leaderboard-table-banner box gradient"><h3>Leaderboard</h3></div>
+</banner>
+<div class="filters">
+  <select class="benchmark-sort">
+    <option value="" disabled selected>Sort by Benchmarks</option>
+    <option value="average">Sort by average</option>
+    {% for name in benchmark_names %}
+      <option value="{{name}}">{{ name }}</option>
+    {% endfor %}
+  </select>
+  <div class="legend">
+    <label>Score Legend</label>
+    <span>No Alignment</span>
+    <span class="legend-right">Alignment at ceiling</span>
+  </div>
+  <div class="leaderboard-breadcrumb"></div>
+</div>
+<div class="leaderboard-table-container">
+  <table class="table">
+    <thead>
+      <tr>
+        <th class="rank-header">
+          <p>Rank</p>
+        </th>
```

Comment:
Again not sure if this is the right spot for this code change, but perhaps the horizontal scroll bar should be right under the rank/all the headers? Right now it is all the way at the bottom, and if we want to scroll to see say behavior we have to scroll all the way down